.github/workflows/ci.yml

```diff
@@ -341,10 +341,8 @@ jobs:
           - "aarch64-linux"
           - "aarch64-darwin"
         test-tags:
-          - "activate"
           - "containerize"
-          - "catalog"
-          - "!activate,!containerize,!catalog"
+          - "!containerize"
```

Comment:
needs to change the Unit tests filter

Reply:
That filter is for the "dev" tests, here we should be reducing the "remote" tests.

Reply:
i see different matrix 👍🏼